static/src/js/zustand/selectors/preferences.js

```diff
@@ -0,0 +1,47 @@
+import { translateDefaultCollectionSortKey } from '../../util/collections'
+
+/**
+ * Get preferences from Zustand store
+ */
+export const getPreferences = (state) => state.preferences.preferences
+
+/**
+ * Get map preferences from Zustand store
+ */
+export const getMapPreferences = (state) => {
+  const preferences = getPreferences(state)
+
+  return preferences?.mapView
+}
+
+/**
+ * Get collection sort preference from Zustand store
+ */
+export const getCollectionSortPreference = (state) => {
+  const preferences = getPreferences(state)
+
+  return preferences?.collectionSort
+}
+
+/**
+ * Get collection sort key parameter with preference comparison
+ * Returns null if the sort key matches user preference (to hide from URL)
+ */
+export const getCollectionSortKeyParameter = (paramCollectionSortKey, state) => {
```

Comment:
selectors shouldn't have more than one parameter, they only receive the zustand state.

Think of these selectors as the exact same function you'd pass into `useEdscStore()`. The easy example is `getPreferences`:
```
useEdscStore((state) => state.preferences.preferences)
// is exactly the same as 
useEdscStore(getPreferences)
```

So you can't pass in paramCollectionSortKey, you have to go get it within the function
```
  const {
    getState: reduxGetState
  } = configureStore()
  const reduxState = reduxGetState() 
  const paramCollectionSortKey = getParamCollectionSortKey(reduxState) // this is the old redux selector
```

Where this is being used in `UrlQueryContainer` needs to be moved out of `mapStateToProps` and into the `useEdscStore` call